### ISSUE TITLE
Usage of the console with "console-less" network participants

### ISSUE THREAD

**Issue by aMinuteMan** [hyperledger-labs/fabric-operations-console#219]
After working with the console for a while I can tell it is a great tool which removes a lot of complexity (creating channels with all possible options, updating them, collecting of signature, etc.) and provides a nicely designed UI.

However, in order to enjoy these benefits all network participants must use the UI, particularly when it comes to sending and receiving signature requests.

This opens the question how the console can be utilized in this environment.

An example would be to update a channel where a network participant using the console and a network participant not using the console have to sign a config update.

My understanding so far is that the consoles communicate directly with each other, which wouldn't work in this scenario. The second network participant would expect a config update block to sign and the first network participant would expect some kind of notification containing at least the signed block plus additional data.

In short, can the console be used in such a scenario? If yes, how, resp. is there any best-practice?

**Comment by dshuffma-ibm** [comment]
Well... it is theoretically possible to coordinate console users with non-console users across signature request events. But it will be exceedingly difficult. Its hard b/c of the auth. Its not the auth/signatures that fabric will check, but the auth that the console will check for before accepting edits/creation of signature collections. 

**For that reason we don't really support what you are after.** However it would be a great improvement if we could. What we need is a way to wrap up the signature creation and formatting in some lib.

If someone is reading this and wants to make it happen, here's the details of some apis where you would start.
- Get signature collections api - https://github.com/hyperledger-labs/fabric-operations-console/blob/main/packages/athena/json_docs/json_validation/ibp_openapi_v3.yaml#L7130
- Edit signature collection api - https://github.com/hyperledger-labs/fabric-operations-console/blob/main/packages/athena/json_docs/json_validation/ibp_openapi_v3.yaml#L7416
- Backend code for validating signatures - https://github.com/hyperledger-labs/fabric-operations-console/blob/main/packages/athena/libs/signature_collection_lib.js#L1325
- Docs/readme on signature collection apis - https://github.com/hyperledger-labs/fabric-operations-console/blob/main/packages/athena/docs/signature_collection_apis.md

**Comment by varadgit** [comment]
It could be a nice to have, but it is not in the roadmap... will could reopen if there are people interested in contributing.
